app/controllers/admin/pet_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Admin::PetApplicationsController < ApplicationController
```

Comment:
Nice work using the Admin namespacing and pulling this out into a Joins controller! This is the most RESTful option!